pkg/framework/operators/operator.go

```diff
@@ -1,14 +1,11 @@
 package operators
 
-import (
-	restclient "k8s.io/client-go/rest"
-)
-
 type OperatorSetupBuilder interface {
-	NewForConfig(namespace string, restConfig *restclient.Config) (OperatorSetup, error)
+	Build() (OperatorAccessor, error)
```

Comment:
@mkrutov How can you make this interface work without any info on the target namespace and rest k8s clients to use?

Reply:
If we provide OperatorBuilder into NewFramework or additional Framework setup, we init it in tests, providing all the info, at least that is my vision for it. This is up for discussion, actually.

Reply:
But if you look into the OperatorSetupBuilder interface in isolation from the "Framework" (loose coupling), it does not have anything that provides where to setup the operator.